components/tidb_query_common/src/storage/scanner.rs

```diff
@@ -103,6 +103,7 @@ impl<T: Storage> RangesScanner<T> {
                 if let Some(r) = self.scanned_rows_per_range.last_mut() {
                     *r += 1;
                 }
+                resource_metering::record_read_keys(1);
```

Comment:
I'm afraid it is a repeat count.

Reply:
My understanding of this is not professional, could you please tell me where duplication happened? Our goal is to make the granularity of the record as fine as possible. (record once immediately when scanning one row)

Reply:
If you trace L79, finally you will get into src/storage/mvcc/reader/point_getter.rs:297
If you trace L88, finally you will get into src/storage/mvcc/reader/scanner/backward.rs or src/storage/mvcc/reader/scanner/forward.rs.

Reply:
This count happens at coprocessor part. Will we miss counts? What about non-coprocessor KV scan requests?

Reply:
> This count happens at coprocessor part. Will we miss counts? What about non-coprocessor KV scan requests?

We don't need to handle coprocessor part. All txn read will get into storage/mvcc/reader, coprocessor included.

Reply:
Thanks for this information~ Removed this line due to duplicate record.